src/LiveQuery/ParseWebSocketServer.js

```diff
@@ -1,12 +1,20 @@
 import logger from '../logger';
 
+/* eslint-disable */
```

Comment:
what are you disabling?

is it just the next line?  if so, use:

// eslint-disable-next-line foo

otherwise, reenable where it makes sense

also put the specific rule you're trying to turn off?